spec/models/code_of_conduct_spec.rb

```diff
@@ -0,0 +1,16 @@
+describe CodeOfConduct do
+  describe "#create_signature_request" do
+    it 'returns and stores a signature request id', :vcr do
+      student = FactoryGirl.create(:student)
+      # Test didn't pass with FactoryGirl, but does otherwise. Has to do with the API callback.
+      code_of_conduct = CodeOfConduct.create(student_id: student.id)
```

Comment:
put comment on line talking about. also, not API callback - after_initialize callback